Jenkinsfile-check-versions

```diff
@@ -0,0 +1,56 @@
+library("tdr-jenkinslib")
+
+pipeline {
+  agent {
+    label "master"
+  }
+  parameters {
+    choice(name: "STAGE", choices: ["intg", "staging", "prod"], description: "The stage you are deploying Terraform changes to")
```

Comment:
What does this parameter do? Doesn't this only run on the mgmt account?

Reply:
Yep, I copied this from another file as you can tell :smile: I've removed this.